src/app/public/src/modules/code-block/code-block.component.scss

```diff
@@ -8,7 +9,6 @@
   border: 1px solid $stache-border-color-neutral-medium;
   border-top: none;
   margin-top: 0;
-  background-color: $stache-background-color-selected;
```

Comment:
We might need to add a `background-color: #fff` if that's the direction we want to go.  This will prevent the background from being transparent on some layouts and cause issues with visibility.  A crude example:
![screen shot 2018-11-13 at 1 33 51 pm](https://user-images.githubusercontent.com/20796436/48434844-e7f42980-e748-11e8-9a20-bac9169d388e.png)

Reply:
As a side note here, swapping the background color to white, I am curious about your thoughts on making the header color itself the old background color, and adjusting the text color to be accessible still.  

I would love to have @Blackbaud-ToddRoberts weigh in on the color choices for this PR

Reply:
![screen shot 2018-11-13 at 1 42 23 pm](https://user-images.githubusercontent.com/20796436/48435257-073f8680-e74a-11e8-8895-365934f945ce.png)
I think the current font color and the `$stache-background-color-selected` pass all but AAA on normal text, which I think is passing for us?

Reply:
Todd and I discussed making the header a different color, but decided on just making the whole component white. I believe that adding color to the header also introduces more contrast issues with the language label, but I'd have to confirm. 

Adding #fff to the background and header is a good idea though, I'll deliver that now.

Reply:
It passes all but AAA normal text with the language label.  Though if UX believes all white is better than a contrast header that's fine.

Reply:
I think the white header has enough distinction from the content due to the border, and is low visual weight to distract from the surrounding content. I liken it to a sort of combo between a tile header and list toobar which both have white bgs